skopt/dummy_opt.py

```diff
@@ -4,16 +4,19 @@
 import inspect
 import numbers
 import numpy as np
+from time import time
 
 from collections import Iterable
 from scipy.optimize import OptimizeResult
 from sklearn.utils import check_random_state
 
 from .space import Space
-
+from .utils import create_result
+from .utils import verbose_func
```

Comment:
It's difficult to think of one. There is quite a bit information to be represented in a single name :( (minimum, time taken and function value obtained).